r10k.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.license  = 'Apache-2.0'
 
   s.add_dependency 'colored',   '1.2'
-  s.add_dependency 'cri',       '~> 2.15.1'
+  s.add_dependency 'cri',       '>= 2.15.1', '< 2.15.4'
```

Comment:
Would you mind adding a comment here about why this was pinned? Ideally we should remove the pin eventually, once we also want to drop Ruby 2.3 support.